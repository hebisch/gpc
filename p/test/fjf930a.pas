{ Modification of scott1.pas for non-standard Pascal modes }

program test(output);

var f: text;
    c: char;

begin

   rewrite(f);
   writeln(f, 'how now');
   writeln(f, 'brown cow');
   reset(f);
   write('''');
   while not eof(f) do begin

      if eoln(f) then write('<eoln>');
      read(f, c);
      write(c)

   end;
   write('''');
   writeln(' s/b ''how now<eoln> brown cow<eoln> ''');
   rewrite(f);
   writeln(f, 'too much');
   write(f, 'too soon');
   reset(f);
   write('''');
   while not eof(f) do begin

      if eoln(f) then write('<eoln>');
      read(f, c);
      write(c)

   end;
   write('''');
   writeln(' s/b ''too much<eoln> too soon''');

end.
