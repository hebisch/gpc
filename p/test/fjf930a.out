'how now<eoln> brown cow<eoln> ' s/b 'how now<eoln> brown cow<eoln> '
'too much<eoln> too soon' s/b 'too much<eoln> too soon'
